src/shm_sub.c

```diff
@@ -2637,9 +2637,10 @@ sr_shmsub_rpc_notify(sr_conn_ctx_t *conn, sr_rwlock_t *sub_lock, off_t *subs, ui
         goto cleanup;
     }
 
-    if (!strcmp(path, SR_RPC_FACTORY_RESET_PATH) && (cur_priority == SR_RPC_FACTORY_RESET_INT_PRIO)) {
-        assert(subscriber_count == 1);
+    /* There can be only 1 RPC subscription at each prioirty level */
+    assert(subscriber_count == 1);
```

Comment:
Why did you change the `assert`? I would prefer for the code to remain unified and work correctly if we decide to support more subscriptions on RPCs for a single priority.

Reply:
Oh okay, I will remove this part of the code. I assumed it is never possible to have more than one subscription at the same priority for RPCs.

Reply:
Your assumption is correct but I prefer for this code to be the same for all subscriptions, if we decide to support it later and simpler maintenance and bug fixing.